contentgrid-spring-data-rest/src/main/java/org/springframework/data/rest/webmvc/DelegatingRepositoryPropertyReferenceController.java

```diff
@@ -283,6 +294,16 @@ public ResponseEntity<Void> handle(
         return exception.toResponse();
     }
 
+    private ResponseEntity<?> fallbackIfRequired(ResponseEntity<?> original, ThrowingSupplier<ResponseEntity<?>> defaultSupplier) throws Exception {
+        if(!contentGridRestProperties.isFallbackToDefaultRelationController()) {
+            return original;
+        }
+        if(original.getStatusCode() == HttpStatus.NOT_IMPLEMENTED) {
+            return defaultSupplier.get();
+        }
+        return original;
+    }
```

Comment:
I feel like the wording could be improved, _default_ and _original_ don't really contrast each other in an obvious way.
What do you think of
```
fallbackIfRequired(ResponseEntity<?> cgResponse, ThrowingSupplier<ResponseEntity<?>> springDRDefaultSupplier) { ... }
```
?

Additionally, I think _fallbackIfRequired_ could be _fallbackIfNotImplemented_ or something like that? "If Required" sounds like it will always pick the second option if the config property is set.